### PR TITLE
Add red text to Copter, Rover, and Plane toc 1st time items

### PR DESCRIPTION
**Pull request by TunaLobster**
Looks a little something like this. Looks very similar on Rover and Copter too.
image

### CODE REVIEW

File: common/source/_static/common_theme_override.css [hunk 1]
```diff
@@ -45,3 +45,14 @@ table.partners-table td {
...
+
+/* make certain toc sidebar items red */
+.wy-menu .toctree-l1 a[href="docs/arduplane-setup.html"],
+.wy-menu .toctree-l1 a[href="docs/first-flight-landing-page.html"],
+.wy-menu .toctree-l1 a[href="docs/initial-setup.html"],
+.wy-menu .toctree-l1 a[href="docs/flying-arducopter.html"],
+.wy-menu .toctree-l1 a[href="docs/apmrover-setup.html"],
+.wy-menu .toctree-l1 a[href="docs/rover-first-drive.html"]
+{
+    color: red;
```
Comment (Hwurzburg) [inline common/source/_static/common_theme_override.css:57]:
```suggestion
    color: red; font-weight: bold;
```
too bad it isnt persistent....also, what about the Quadplane TOC entries? can they be made to do the same thing?

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
thanks

File: common/source/_static/common_theme_override.css (hunk 1 above)
Comment (TunaLobster) [reply common/source/_static/common_theme_override.css:57]: Yep! Just need to add the correct CSS class and attribute things. What do you mean by persistent?

Comment (Hwurzburg) [reply common/source/_static/common_theme_override.css:57]: if you click on another menu item, and then return (ie close that items expansion) the style reverts....its only red until you click something....not a biggie

Comment (TunaLobster) [reply common/source/_static/common_theme_override.css:57]: Fixed both

**Comment by Hwurzburg** [comment]
playing with it some more, it seems to eventually refresh...thanks merging

**Pull request merged**